api/python/quilt3/util.py

```diff
@@ -25,6 +25,7 @@
 OPEN_DATA_URL = "https://open.quiltdata.com"
 
 PACKAGE_NAME_FORMAT = r"([\w-]+/[\w-]+)(?:/(.+))?$"
+DISABLE_TQDM = os.getenv('QUILT_USE_TQDM', '').lower() != 'true'
```

Comment:
@akarve, this disables `tqdm` by default, is it intended?
Also name `QUILT_USE_TQDM` might be too specific, IMHO `QUILT_INTERACTIVE` or `QUILT_PROGRESS_BARS` or something might be better.

Reply:
No, disabling `TQDM` by default is not intended. @cosmic-byte please make default "on".
And let's go with `QUILT_MINIMIZE_STDOUT` and not in the docs that this setting is useful for logging. @cosmic-byte please document this here somewhere https://docs.quiltdata.com/api-reference/api, there should be a section on "Environment variables"